core/src/main/java/org/elasticsearch/index/query/Operator.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.elasticsearch.index.query;
 
+import org.apache.lucene.queryparser.classic.QueryParser;
```

Comment:
instead of cutting this over can we just use the lucene enum and use the ordinal to serialize?